api/genomics/src/test/java/org/pmiops/workbench/genomics/RandomizeVcfTest.java

```diff
@@ -0,0 +1,63 @@
+package org.pmiops.workbench.genomics;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFFileReader;
+import java.io.File;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class RandomizeVcfTest {
+  private static final VCFFileReader reader =
+      new VCFFileReader(new File("src/test/resources/NA12878_204126160130_R01C01.toy.vcf.gz"));
+  private static final VariantContext variantContext = reader.iterator().next();
+  private static final RandomizeVcf randomizeVcf = new RandomizeVcf();
+
+  @BeforeClass
+  public static void setUp() {
+    randomizeVcf.sampleNameSuffix = "0";
+  }
+
+  @Test
+  public void TestRandomizeVariant() {
+    VariantContext randomizedVariant = randomizeVcf.randomizeVariant(variantContext);
+    // For each sample/GT in the variant...
+    randomizedVariant
+        .getGenotypes()
```

Comment:
opt nit: you could probably reduce some nesting here by using stream `flatMap`. This basically lets you expand the current stream with more elements, i.e. map a small collection of genotypes into a larger collection alleles

```
.getGenotypes()
.stream()
.flatMap(Genotype::getAlleles)
.forEach(... assertion ...)
```

Reply:
Oh yeah, that's true, thank you